pkg/trace/testutil/stats_test.go

```diff
@@ -8,12 +8,16 @@ package testutil
 import (
 	"testing"
 
+	"github.com/DataDog/datadog-agent/pkg/trace/config"
 	"github.com/stretchr/testify/assert"
 )
 
+var conf = &config.AgentConfig{}
+
 func TestRandomBucket(t *testing.T) {
+
 	for i := 10; i < 100; i += 10 {
-		b := RandomBucket(i)
+		b := RandomBucket(i, conf)
```

Comment:
OK I see where we're using `RandomBucket`. I don't think we need to change this test. But please tell me if I'm missing something.

Reply:
So the function RandomBucket calls BucketWithSpans, which then calls HandleSpan. HandleSpan needs the agent config, so I passed the config from RandomBucket as a parameter and passed it down. Because of this, all the tests that use RandomBucket need the agent config as a parameter too or it throws an error. Not sure if there's a better approach for this case, let me know if you have suggestions!